ggshield/scan/scannable_errors.py

```diff
@@ -43,3 +43,10 @@ def handle_scan_error(detail: Detail, chunk: List[Dict[str, str]]) -> None:
         )
 
         display_error(str(detail))
+
+
+def handle_scan_error(detail: Detail) -> None:
+    if detail.status_code == 401:
+        raise click.UsageError(detail.detail)
+    display_error("\nError scanning. Results may be incomplete.")
```

Comment:
Will there ever be any result ?

If not, the `Results may be incomplete.` can be removed I think.